sklearn/preprocessing/tests/test_data.py

```diff
@@ -2303,6 +2303,35 @@ def test_power_transformer_lambda_zero():
     assert_array_almost_equal(pt.inverse_transform(X_trans), X)
 
 
+def test_standard_scaler_with_std():
+    rng = np.random.RandomState(0)
+    X = rng.randint(0, 2, (100, 2))
+    scaler = StandardScaler(copy=True, with_mean=True, with_std=2)
+    X_scaled = scaler.fit(X).transform(X, copy=True)
+    assert np.isclose(X_scaled.mean(), 0.0)
+    assert np.isclose(X_scaled.std(), 0.5)
+    scaler = StandardScaler(copy=True, with_mean=True, with_std=True)
+    X_scaled = scaler.fit(X).transform(X, copy=True)
+    assert np.isclose(X_scaled.mean(), 0.0)
+    assert np.isclose(X_scaled.std(), 1.0)
+
+
+@pytest.mark.parametrize('with_std', [-10, -1, 3, 10])
+def test_scale_with_std(with_std):
```

Comment:
Please split this tests into two test:

1. `test_scale_with_std()` to just test `scale`
2. `test_scale_with_std_error(with_std)` to test the `ValueError`. In `pytest.raises` uses the `match` keyword to assert the text of the error message.

Reply:
Done